pydantic_ai_slim/pydantic_ai/_agent_graph.py

```diff
@@ -915,8 +915,10 @@ async def handle_call_or_result(
                     yield event
 
         else:
+            # TODO(Marcelo): We need to replace this with `anyio.as_completed()` from
+            # https://github.com/agronholm/anyio/pull/890.
```

Comment:
You intend to vendor that function right? Because I don't think we should wait until that's released and then bump the requirement all the way to the latest -- that'd be annoying for users